requirements.txt

```
@@ -1,4 +1,4 @@
-sphinx~=3.5
+sphinx
```

```suggestion
sphinx>=4.0.0
```

I'd suggest explicitly telling the dependency resolver that if everything else is unconstrained then you need the Sphinx v4.0 API. I've tested this with `~=4.0.0` and `>=4.0.0`.